### PR TITLE
CONSOLE-3303: Move plugin template to the openshift GitHub org - add artifects for screenshots and videos

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
Adding the artifects for screenshots and videos to help figure out  the [failing tests](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/33499/rehearse-33499-pull-ci-openshift-console-plugin-template-main-e2e-aws-plugin-template/1646654396903198720) issue and verify if the `console-plugin-template` installation is successfully in CI env.

### CODE REVIEW

File: test-prow-e2e.sh [hunk 1]
```diff
@@ -7,9 +7,38 @@ SCREENSHOTS_DIR=integration-tests/screenshots
...
+  oc get subscriptions -n ${NS} -o yaml >> ${ARTIFACT_DIR}/subscription_details.yaml
+  oc get csvs -n ${NS} -o wide > ${ARTIFACT_DIR}/csvs.yaml
+  oc get csvs -n ${NS} -o yaml >> ${ARTIFACT_DIR}/csvs.yaml
+  oc get deployments -n ${NS} -o wide > ${ARTIFACT_DIR}/deployment_details.yaml
+  oc get deployments -n ${NS} -o yaml >> ${ARTIFACT_DIR}/deployment_details.yaml
+  oc get installplan -n ${NS} -o wide > ${ARTIFACT_DIR}/installplan.yaml
+  oc get installplan -n ${NS} -o yaml >> ${ARTIFACT_DIR}/installplan.yaml
+  oc get nodes -o wide > ${ARTIFACT_DIR}/node.yaml
+  oc get nodes -o yaml >> ${ARTIFACT_DIR}/node.yaml
+  oc get pods -n ${NS} -o wide >> ${ARTIFACT_DIR}/pod_details_openshift-kubevirt.yaml
```
Comment (jhadvig) [inline test-prow-e2e.sh:25]: why for the kubevirt pod ?

Comment (cyril-ui-developer) [reply test-prow-e2e.sh]: @jhadvig Remove.

File: test-prow-e2e.sh [hunk 2]
```diff
@@ -7,9 +7,38 @@ SCREENSHOTS_DIR=integration-tests/screenshots
 INSTALLER_DIR=${INSTALLER_DIR:=${ARTIFACT_DIR}/installer}
 
 function copyArtifacts {
+  oc cluster-info dump > ${ARTIFACT_DIR}/cluster_info.json
```
[the remark this answers is not in the document]
Comment (cyril-ui-developer) [reply test-prow-e2e.sh:10]: > why do we need these ?

I think we need this to get more info about the cluster.

**Comment by jhadvig** [comment]
@cyril-ui-developer is not really clear to me why we need all those resources ?

**Comment by cyril-ui-developer** [comment]
> @cyril-ui-developer is not really clear to me why we need all those resources ?

@jhadvig To verify if the plugin-template is installed before the test runs

**Pull request merged**